doc/manual.asciidoc

```diff
@@ -190,14 +190,23 @@ you don't need to pass `-j`.)
 GNU Jobserver support
 ~~~~~~~~~~~~~~~~~~~~~
 
-Since version 1.13., Ninja builds can follow the
+Since version 1.14., Ninja builds can follow the
 https://https://www.gnu.org/software/make/manual/html_node/Job-Slots.html[GNU Make jobserver]
-client protocol. This is useful when Ninja is invoked as part of a larger
-build system controlled by a top-level GNU Make instance, or any other
-jobserver pool implementation, as it allows better coordination between
-concurrent build tasks.
+protocol. It supports both the client and pool modes.
 
-This feature is automatically enabled under the following conditions:
+Client mode is useful when Ninja is invoked as part of a larger
+build system controlled by a top-level Ninja or GNU Make instance, or
+any other jobserver pool implementation, as it allows better coordination
+between concurrent build tasks.
+
+Pool mode is useful when Ninja is the top-level build tool that
+invokes sub-builds recursively in a similar setup.
+
+To enable pool mode, use `--jobserver-pool` on the command line. This also
+enables client mode.
```

Comment:
It is trivially knowable when build.ninja is written, whether pool mode is useful. So it would be nice if we didn't need to rely on a CLI flag to enable it.

Reply:
This seems to work, so I uploaded a new PR with a new commit to implement this specifically. Looking for @jhasse's opinion on this though.